netbox/circuits/api/nested_serializers.py

```diff
@@ -47,8 +48,9 @@ class Meta:
 
 class NestedCircuitTerminationSerializer(WritableNestedSerializer):
```

Comment:
`NestedCircuitTerminationSerializer` is used by other views. We cannot modify it arbitrarily. Extending the `CircuitSerializer` to include a circuit's terminations will likely require the introduction of a new serializer specifically for this endpoint.

Reply:
> `NestedCircuitTerminationSerializer` is used by other views.
Really? I didn't find any places where it was referenced. Had I known it was used, I wouldn't have removed anything from it to avoid breaking existing functionality (i.e. only addition of fields).

Reply:
It unfortunately escapes IDE attribution, but it is used dynamically by CableSerializer (to represent a cable connected to a CircuitTermination). It may be used elsewhere.

Reply:
Aha! Good catch. Would you prefer that I restore the removed fields and have it so this same serializer is shared between the two, or just create another one as you suggested? Now that I know it's already used, I'd go with the latter just to be safe.

Reply:
Best to create a new one. Silly as it sounds, you should name it `CircuitCircuitTerminationSerializer`. We've had to do this in other places as well, such as `IPAddressNestedInterfaceSerializer`.

Reply:
PR updated accordingly.